ample/main.py

```diff
@@ -422,8 +422,9 @@ def monitor():
             submit_max_array=optd['submit_max_array'])
 
         if not ok:
-            msg = "Error running MRBUMP on the ensembles!\nCheck logs in directory: {0}".format(optd['mrbump_dir'])
-            exit_util.exit_error(msg)
+            msg = "An error code was returned after running MRBUMP on the ensembles!\n" + \
+                  "For further information check the logs in directory: {0}".format(optd['mrbump_dir'])
+            logger.critical(msg)
 
```

Comment:
Does that mean we change behaviour and do not crash out when an ensemble fails?

Reply:
Yup - crashing a whole job just because (potentially) a single search model failed and returned an error code was a bad idea.